src/commands/shared.ts

```diff
@@ -1,5 +1,14 @@
 import { Argv } from "yargs";
 
+export enum Network {
+  "mainnet" = 1,
+  "ropsten" = 3,
+  "rinkeby" = 4,
+  "goerli" = 5,
+  "local" = 1337,
+}
```

Comment:
Nice! Thanks for adding that!👍

Shall we add Polygon (137) and Polygon Mumbai (80001) to the options too?